### ISSUE TITLE
Error uploading weight to Garmin Connect: forbidden

### ISSUE THREAD

**Issue by Martin22** [davidkroell/bodycomposition#19]
Hi. Since this morning, it is not possible to import data into Garmin Connect, the import is reporting an error "forbidden".

**Comment by JoeBee01** [comment]
Same Problem here. Also starting this morning. Seems to be general issue

**Comment by RobertWojtowicz** [comment]
Looks like it, I have the same thing. let's wait a while maybe a failure at Garmin again: https://github.com/petergardfjall/garminexport/issues/103

**Comment by davidkroell** [comment]
Yes is a general issue, I'll wait for abrander/garmin-connect/issues/26 to be fixed.

**Comment by saborsoft** [comment]
Do you have any update on it? I checked it today and still got this error.

**Comment by davidkroell** [comment]
Unfortunately not. Garmin is actively trying to mitigate imports from third party API usages. You now have to be approved to use the API.

Therefore it's unclear for now whether this will continue to work ever again.

Sorry for inconvenience but Garmin is not on our side on this problem.

**Comment by saborsoft** [comment]
Totally understand, thanks for your quick answer!

This part was unclear for me, so not me as a user can approve the api login to my account, but garmin needs to approve a login to a third party api call in general, like here to the bodycomposition app, right?

**Comment by RobertWojtowicz** [comment]
@Martin22 @JoeBee01 @saborsoft check out new version :)
https://github.com/RobertWojtowicz/miscale2garmin

BR,
Robert
